docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -594,3 +638,20 @@ in following shell scripts:
 - `$ZOWE_RUNTIME/api-mediation/scripts/api-mediation-start-catalog.sh`
 - `$ZOWE_RUNTIME/api-mediation/scripts/api-mediation-start-discovery.sh`
 - `$ZOWE_RUNTIME/api-mediation/scripts/api-mediation-start-gateway.sh`
+
+## Security Service Client library
+
+The `security-service-client-spring` library contains authentication providers and filters to handle the authentication against the z/OSMF service. 
+There are two providers:
+
+1. The `Gateway Login Provider` that verifies credentials against z/OSMF service
+2. The `Gateway Token Provider` that authenticate the JWT token provided by z/OSMF
+
+The library also contains three Spring Security filters:
+
+1. The `Basic Content Filter` that authenticates the credentials from the basic authorization header
+2. The `Cookie Content Filter` that authenticates the JWT token stored in the cookie by extracting the username and the JWT token from them
+3. The `Login Filter` that processes the authentication requests with the username and password in a JSON format.
+
+This component can be used by any client which needs authentication provider against z/OSMF. 
```

Comment:
Any client can use this component that requires authentication provider against z/OSMF.